docs/releases.md

```diff
@@ -0,0 +1,65 @@
+# Release process
+
+The release process in broken up into 3 steps.
+
+1. Increment the version and add a new tag
+2. Modify the changelog
+3. Merge the release branch and create a "github release"
+
+## 1. Increment the version and add a new tag
+
+Run
+
+```shell
+$ yarn version
+```
+
+See [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200).
```

Comment:
Can you revise this to make it obvious _why_ this link matters. Something like "The version number should be updated following [semantic versioning](...)". I think you can actually omit the list below on the basis that most readers will likely already know what the semver rules.